src/internal/pubsub-client.ts

```diff
@@ -108,7 +116,8 @@ export class PubsubClient {
   ): Promise<TopicPublish.Response> {
     try {
       validateCacheName(cacheName);
-      // todo: validate topic name
+      validateTopicName(topicName);
+      validateTopicValue(value);
```

Comment:
I'm not sure we want to disallow sending an empty message? It doesn't look to me like the golang client is doing a similar validation.

Reply:
I was unaware we were allowing empty strings. Reverted in https://github.com/momentohq/client-sdk-nodejs/pull/356/commits/eee047d1d07d0487b64c8d7a5ea34a81a1bdab5a